docs/getting-started/installation.mdx

```diff
@@ -48,8 +48,25 @@ npm install next-transpile-modules --save-dev
 Then, add this to your `next.config.js`
 
 ```js
-const withTM = require('next-transpile-modules')(['three'])
-module.exports = withTM()
+const withTM = require("next-transpile-modules")([
+    "three",
+    "@react-three/fiber",
+    "@react-three/drei",
+  ]);
+  
+  module.exports = withTM({
+    webpack(config, options) {
+      config.module.rules.push({
+        test: /\.(glb|gltf)$/,
+        use: {
+          loader: "file-loader",
```

Comment:
And then this again, I've never had to do this before.

Reply:
Yes, you should be typically have them in `public` and then import them from root like:

```js
useGLTFLoader('/models/my-model.glb')
```